Processes/How-To-Write-Content-for-the-Docs-Portal.md

```diff
@@ -9,4 +9,4 @@ Provide content to the DevRel team in Markdown (.md) format. Please use our Mark
 
       * Use active voice. Refer to the [Google Style Guide](https://github.com/google/styleguide), as needed.
 
-7. [Submit your content](./How-To-Submit-Content-to-the-Docs-Portal.md) via PR
+3. [Submit your content](./How-To-Submit-Content-to-the-Docs-Portal.md) via PR
```

Comment:
via PR -> via a Pull Request (PR).

For consistency with the how-to-submit... doc, add a period (.) at the end of every action numbered bullet, and sub-bullets (for 1. 2. 3. and * * inside).

Reply:
Done